### PR TITLE
update publications

### PR DESCRIPTION
**Pull request by tjhei**
(Sorry for the change in order of the entries. They should be sorted by "date added" from now on)

### CODE REVIEW

**Comment by tjhei** [comment]
hang on, there are no new entries in there...
related to  #2762

**Comment by tjhei** [comment]
updated. Check the second commit for the new entries. The first one only reorders them.

File: doc/manual/citing_aspect.bib [hunk 1]
```diff
@@ -306,3 +321,67 @@ @techreport{Bangerth2017a
 url = {https://doi.org/10.6084/m9.figshare.4865333},
 year = {2017}
 }
+@article{Bredow2018a,
```
Comment (gassmoeller) [inline doc/manual/citing_aspect.bib]: I think I accidentally duplicated this entry in the Mendeley group and now it appears twice in the bib file. I removed the duplicate copy from Mendeley, could you do the same here? Sorry for that.

File: doc/manual/citing_aspect.bib [hunk 2]
```diff
@@ -306,3 +321,67 @@ @techreport{Bangerth2017a
...
+year = {2018}
+}
+@article{Heron2018,
+author = {Heron, P J and Pysklywec, R N and Stephenson, R and van Hunen, J},
+journal = {Geology},
+publisher = {Geological Society of America},
+title = {{Deformation driven by deep and distant structures: Influence of a mantle lithosphere suture in the Ouachita orogeny, southeastern United States}},
+year = {2018}
+}
+@article{Bredow,
```
Comment (gassmoeller) [inline doc/manual/citing_aspect.bib]: This is actually a thesis not an article.

**Comment by tjhei** [comment]
:+1:

**Pull request merged**